automation/test.sh

```diff
@@ -372,27 +372,33 @@ if [[ -z ${KUBEVIRT_E2E_FOCUS} && -z ${KUBEVIRT_E2E_SKIP} ]]; then
     export KUBEVIRT_E2E_FOCUS="\\[sig-storage\\]|\\[storage-req\\]"
     export KUBEVIRT_E2E_SKIP="Migration"
   elif [[ $TARGET =~ vgpu.* ]]; then
-    export KUBEVIRT_E2E_FOCUS=MediatedDevices
+    export KUBEVIRT_E2E_FOCUS="MediatedDevices"
   elif [[ $TARGET =~ sig-compute-realtime ]]; then
     export KUBEVIRT_E2E_FOCUS="\\[sig-compute-realtime\\]"
   elif [[ $TARGET =~ sig-compute-migrations ]]; then
     export KUBEVIRT_E2E_FOCUS="Migration"
-    export KUBEVIRT_E2E_SKIP="GPU|MediatedDevices"
+    export KUBEVIRT_E2E_SKIP="MediatedDevices"
+    export ginko_params="$ginko_params --label-filter=\"!needs-gpu\""
   elif [[ $TARGET =~ sig-compute ]]; then
     export KUBEVIRT_E2E_FOCUS="\\[sig-compute\\]"
-    export KUBEVIRT_E2E_SKIP="GPU|MediatedDevices|Migration"
+    export KUBEVIRT_E2E_SKIP="MediatedDevices|Migration"
+    export ginko_params="$ginko_params --label-filter=\"!needs-gpu\""
   elif [[ $TARGET =~ sig-monitoring ]]; then
       export KUBEVIRT_E2E_FOCUS="\\[sig-monitoring\\]"
   elif [[ $TARGET =~ sig-operator ]]; then
     export KUBEVIRT_E2E_FOCUS="\\[sig-operator\\]"
   elif [[ $TARGET =~ sriov.* ]]; then
-    export KUBEVIRT_E2E_FOCUS=SRIOV
+    export KUBEVIRT_E2E_FOCUS="SRIOV"
   elif [[ $TARGET =~ gpu.* ]]; then
-    export KUBEVIRT_E2E_FOCUS=GPU
+    export KUBEVIRT_E2E_FOCUS=""
```

Comment:
What is the reason for setting this env variable now?

Reply:
@EdDev I had an understanding that since it is being used in _/kubevirt/hack/functests.sh_ , so it required to initialize it (that is why I kept it as an empty string). But I think that initializing **$KUBEVIRT_E2E_FOCUS** is not compulsory, I would remove this env variable as indicated by you above. Thanks for bring that up!